recipe/bld.bat

```diff
@@ -1,8 +1,12 @@
 mkdir build && cd build
 
-cmake .. -G "NMake Makefiles"                  ^
-	-DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%    ^
-	-DCMAKE_BUILD_TYPE=Release                 ^
-	-DBUILD_SHARED_LIBS=OFF
+cmake .. -G "%CMAKE_GENERATOR%" ^
+	-DBUILD_SHARED_LIBS=ON ^
+	-DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+	-DENABLE_ZLIB_COMPRESSION=ON ^
+	-DCMAKE_BUILD_TYPE=Release ^
+	-DENABLE_DEBUG_LOGGING=ON
```

Comment:
Debug builds on Windows actually use different runtime libraries, which makes them incompatible with non-debug built libraries. Not entirely sure if that is what this argument is doing or not.

That said (and this may be off topic), many users have mentioned wanted debug builds on other libraries and there are various scattered discussions on the topic. Maybe we should open an issue on the webpage repo to draw these discussions together and solicit more feedback on how users would like to use these. Thoughts?